src/strange_poker.c

```diff
@@ -15,7 +16,9 @@ int main(int argc, char *argv[])
         fprintf(stderr, "cannot crate ui system\n");
         return -1;
     }
-
+    srand((unsigned int) time(NULL));
+    int *card=malloc(sizeof(int)*52);
```

Comment:
why char card[52] and not int?  you need to empty the array every round anytime so i thought it wouldnt matter whether its dynamic or an array but i can switch it back

Reply:
it could be int but you use it to store only 0 or 1 so char would be enough. Dynamic allocation would be alright but, as I said, you have to do checks if malloc actually allocated memory (`card != NULL`) and handle such situations without leaking resources, so it would be just more work to allocate dynamically.